### ISSUE TITLE
line breaks in HTML blocks

### ISSUE THREAD

**Issue by alissit** [csalzano/gravityforms-html-fields-merge-tags#3]
Great plugin!  Thank you.
I'm finding that when your plugin is activating, additional <br> line breaks are being added to my HTML blocks.
When I disable your plugin, the line breaks disappear.

I usually add HTML to the block like this - and this is causing the additional line breaks:

<h>LOAN AGREEMENT</h>
<p1>THIS AGREEMENT is made as a deed on the date of the last signature below (the Agreement)</p1>

If I add the HTML to the blocks without any formatting like this, then the line breaks are not happening:
`<h2>LOAN AGREEMENT</h2><p>THIS AGREEMENT is made as a deed on the date of the last signature below (the Agreement)</p>`

So I can't drop to a new line when writing out the HTML otherwise a break will happen.

For the sake of my sanity and eyes - I can't write HTMl without line breaks! - could you have a look at the issue?
I'm sure it's something to do with sanitising, but I can't quite figure it out.

I can send you a link to see what I mean if you'd like?

**Comment by csalzano** [comment]
To all HTML fields or only HTML fields where merge tags are used?

**Comment by csalzano** [comment]
Can confirm, the method GFCommon::replace_variables() is adding `<br />` elements. Here is a paste from my debug console:

```$field_content
"<ol><li>(1) iPad</li>
<li>(1) iPad stand</li>
<li>(1) iPad charger</li>
<li>(1) Cash drawer</li>
<li>(1) Receipt printer</li>
<li>(1) Receipt printer cable</li>
<li>Receipt printer paper</li>
<li>Meadow’s full POS software suite</li>
</ul>
"
GFCommon::replace_variables( $field_content, $form, $lead )
"<ol><li>(1) iPad</li><br />
<li>(1) iPad stand</li><br />
<li>(1) iPad charger</li><br />
<li>(1) Cash drawer</li><br />
<li>(1) Receipt printer</li><br />
<li>(1) Receipt printer cable</li><br />
<li>Receipt printer paper</li><br />
<li>Meadow’s full POS software suite</li><br />
</ul><br />"
```

**Comment by csalzano** [comment]
Fixed in 1.0.2
